docker-compose.client.yml

```diff
@@ -1,34 +1,18 @@
 version: "2"
 
 services:
-  client:
-    build: ./client
-    volumes:
-      - "${WORKSPACE:-./workspace}:/usr/src/app/workspace:ro"
-      - "${REPORTS:-./workspace/reports}:/usr/src/app/reports:rw"
-      - "${SUITES:-./suites}:/usr/src/app/suites:ro"
-    environment:
-      - WORKER_PORT # allow this env var to be used in config.js
-      - BALENACLOUD_API_KEY # allow this env var to be used in config.js
-      - BALENACLOUD_ORG # allow this env var to be used in config.js
-      - BALENACLOUD_APP_NAME # allow this env var to be used in config.js
-      - DEVICE_TYPE # allow this env var to be used in config.js
-      - WORKER_TYPE # allow this env var to be used in config.js
```

Comment:
How can we still use these in config.js as env vars if we don't pass them to the parser?